envs/example/standard/hosts

```diff
@@ -10,9 +10,16 @@ controller2
 controller1
 controller2
 
+[mongo_db]
+controller1
+controller2
+
 [db_arbiter]
 compute1
 
+[mongo_arbiter]
```

Comment:
Would it make sense to re-use the generic `db` and `db_arbiter` host groups for mongodb? We're going by the same rules, the db active/active on the controllers and passive on a compute.